test/GithubApi.Issue.test.js

```diff
@@ -0,0 +1,77 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+const path = 'https://api.github.com/user';
+const token = process.env.ACCESS_TOKEN;
+
+let response;
+let reposCount = false;
+let repositories;
+let myRepo;
+let issue;
+let pathIssue;
+
+const myIssue = {
+  title: 'Found a bug'
+};
+
+const bodyIssue = {
+  body: 'Im having a problem with this.'
+};
+
+const instance = axios.create({
+  baseURL: path,
+  headers: { Authorization: `token ${token}` }
+});
+
+describe('Post and patch methods', () => {
+  before(async () => {
+    response = await instance.get(`${path}`);
+    if (response.data.public_repos > 0) {
+      reposCount = true;
+    }
+  });
+
+  it('Verify the service', () => {
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(reposCount).to.equal(true);
+  });
+
+  describe('Verify the list of repositories', () => {
+    before(async () => {
+      repositories = await instance.get(`${response.data.repos_url}`);
+      myRepo = repositories.data.find(({ name }) => name === 'workshop-api-testing-js');
+    });
+
+    it('Repository Content Verification', () => {
+      expect(repositories.status).to.equal(StatusCodes.OK);
+      expect(myRepo).not.to.equal(null);
```

Comment:
as find returns `undefined` If no values satisfy the condition, you should change this assertion to use undefined instead of null

Reply:
resolved

![image](https://user-images.githubusercontent.com/33491290/176551959-23650844-4c46-4420-9ba1-770028f48a7f.png)

Reply:
resolved